main.go

```diff
@@ -391,3 +389,195 @@ func cheap7Estimator(tx []byte) float64 {
 func cheap8Estimator(tx []byte) float64 {
 	return cheapEstimator(tx, 8, 16)
 }
+
+// zlibBatchEstimator simulates a zlib compressor at max compression that works on (large) tx
+// batches.  Should bootstrap it before use by calling it on several samples of representative
```

Comment:
did you want to update this comment? Looks like you don't want bootstrapping in this version where (I think) you're trying to simulate 128kb size batches?

Reply:
the comment is still applicable... we need to bootstrap to at least fill up some of the buffer before we start returning responses